test/metabase/api/search_test.clj

```diff
@@ -193,6 +194,7 @@
              [:like (hsql/call :lower :table_schema) "%foo%"] 0
              [:like (hsql/call :lower :table_name) "%foo%"] 0
              [:like (hsql/call :lower :table_description) "%foo%"] 0
+             [:like (hsql/call :lower :initial_sync_status) "%foo%"] 0
```

Comment:
I think this is incorrect. We do not want to search on initial sync status right? If you search for "complete" it will list every database. We want to show this information, not search on it.

Reply:
ah this is the order clause. I don't think complete should be in here. in `search/order-clause` there's a clause that removes irrelevant fields `(remove #{:collection_authority_level :moderated_status})` and we probably want to add the initial_sync_status to that.